### PR TITLE
An202 address suffix

### PR DESCRIPTION
**Pull request by acenolaza**
Fixes:
1. added "Address" suffix to `airnode`, `sponsor`, `requester` and `sponsorWallet` admin cli args (also renamed internal args and var names)
2. renamed `sponsorWallet` to `sponsorWalletAddress` in node package.

Tests:
1. admin e2e tests passes locally
2. all unit tests passes locally 
3. was able to execute requests via operation package

### CODE REVIEW

**Review by andreogle — APPROVED** [review]

**Comment by acenolaza** [comment]
I've just rebased this branch with latest changes from origin/beta-protocol-revision. I'll merge after CI finishes running the tests.

**Comment by Siegrift** [comment]
Could you wait with merging this and merge after the rebase is merged to
master? :)

**Review by Siegrift — APPROVED** [review]
:+1: :1st_place_medal: I checked code and didn't find anything you missed :)

**Pull request merged**
